interpreter/code_interpreters/language_map.py

```diff
@@ -0,0 +1,17 @@
+from .languages.python import Python
+from .languages.shell import Shell
+from .languages.javascript import JavaScript
+from .languages.html import HTML
+from .languages.applescript import AppleScript
+from .languages.r import R
+
+
+language_map = {
```

Comment:
I ended up needing to reference the same language map that the `create_code_interpreter` module was using, so I moved it into it's own module to make it easier to share.